### PR TITLE
Add document of extracting http path from context

### PR DESCRIPTION
**Pull request by 0daryo**
#### References to other Issues or PRs
Add documentation about #2195 

#### Have you read the [Contributing Guidelines](https://github.com/grpc-ecosystem/grpc-gateway/blob/master/CONTRIBUTING.md)?
Yes.
#### Brief description of what is fixed or changed
Adding documentation of PR #2195.
How to define and extract data from annotated context.

### CODE REVIEW

File: docs/docs/operations/annotated_context.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: AnnotatedContext
+nav_order: 4
+parent: Operations
+---
+
+# AnnotatedContext
```
Comment (johanbrandhorst) [inline docs/docs/operations/annotated_context.md:8]:
```suggestion
# Extracting the HTTP path pattern for a request

It is often interesting to know what [HTTP path pattern](https://github.com/googleapis/googleapis/blob/869d32e2f0af2748ab530646053b23a2b80d9ca5/google/api/http.proto#L61-L87) was matched for a specific request, for example for metrics. This article explains how to extract the HTTP path pattern from the request context.
```

File: docs/docs/operations/annotated_context.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## Get HTTP Path pattern
+1. Define HTTP path in proto option like below, following template ```google.api.http```.
```
Comment (johanbrandhorst) [inline docs/docs/operations/annotated_context.md:11]:
```suggestion
1. Define the HTTP path in the proto annotation. For example:
```

File: docs/docs/operations/annotated_context.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  }
+}
+
+message LoginRequest {}
+
+message LoginReply {}
+```
+
+2. Get values from annotated context e.g. in WithMetadata function.
+You can pass data to backend by adding them to metadata or push them to metrics server.
```
Comment (johanbrandhorst) [inline docs/docs/operations/annotated_context.md:35]:
```suggestion
2. At runtime, get the HTTP path pattern from the annotated context, for example using the `WithMetadata` function.
You can pass data to your backend by adding them to the gRPC metadata or push them to a metrics server.
```

**Review by johanbrandhorst — CHANGES_REQUESTED** [review]
Great start, I made some fixes and suggestions!

**Comment by 0daryo** [comment]
@johanbrandhorst
Thank you for reviewing so kindly!
I just fixed docs.

**Review by johanbrandhorst — APPROVED** [review]

**Comment by johanbrandhorst** [comment]
Thanks for your contribution!

**Pull request merged**